src/test/groovy/org/gradle/android/CrossVersionOutcomeAndRelocationTest.groovy

```diff
@@ -48,13 +48,13 @@ class CrossVersionOutcomeAndRelocationTest extends AbstractTest {
 
         withGradleVersion(gradleVersion.version)
             .withProjectDir(originalDir)
-            .withArguments("assemble", "--build-cache", "--stacktrace")
+            .withArguments("assemble", "createFullJarDebug", "createFullJarDebug", "--build-cache", "--stacktrace")
```

Comment:
I think this should be `"createFullJarDebug", "createFullJarRelease"`

Reply:
The evil copy/paste striked again 😅 thanks for spotting!